opencl/gen_babeltrace_cl_model.rb

```diff
@@ -92,6 +92,47 @@ def parse_field(field)
   { name: name, payload: payload_fields }
 }
 
+environment = [
+  {
+    name: 'hostname',
+    class: 'string',
+  }
+]
+
+packet_context = [
+  {
+    name: 'cpu_id',
+    class: 'unsigned',
+    cast_type: 'uint64_t',
+    class_properties: {
+      field_value_range: 32
+    }
+  }
+]
+
+common_context = [
+  {
+    name: 'vpid',
+    class: 'signed',
+    cast_type: 'int64_t',
+    class_properties: {
+      field_value_range: 64,
+    }
+  },
+  {
+    name: 'vtid',
+    class: 'unsigned',
+    cast_type: 'uint64_t',
+    class_properties: {
+      field_value_range: 64,
+    }
+  }
+]
+
```

Comment:
It is not clear to me why they are now required where they were not before.

Reply:
I thought it was related to btx conversion, all the other backends have them. But does opencl not have them?